### PR TITLE
Add h75-singa ver.

### PR DESCRIPTION
**Pull request by hineybush**
## Description

Add the h75_singa PCB - a 75% keyboard designed for the Singa v3.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by drashna** [comment]
I know you made a comment on discord about this, but your PR looks clean AF! :)

**Comment by hineybush** [comment]
> I know you made a comment on discord about this, but your PR looks clean AF! :)

I might finally be learning..

File: keyboards/hineybush/h75_singa/h75_singa.c [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    return process_record_user(keycode, record);
+}
+
+bool led_update_kb(led_t led_state) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+
+    return led_update_user(led_state);
+}
+*/
```
Comment (fauxpark) [inline keyboards/hineybush/h75_singa/h75_singa.c]: All of this can go.

File: keyboards/hineybush/h75_singa/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h75_singa/keymaps/default/config.h]: This file can be deleted.

File: keyboards/hineybush/h75_singa/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, RESET,   KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,RGB_MOD,
+    KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+    KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+    KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS
+    ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    return true;
+}
```
Comment (fauxpark) [inline keyboards/hineybush/h75_singa/keymaps/default/keymap.c]: This function does nothing and should be removed.

File: keyboards/hineybush/h75_singa/keymaps/wkl_std/config.h [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 hineybush
```
Comment (fauxpark) [inline keyboards/hineybush/h75_singa/keymaps/wkl_std/config.h]: As above.

File: keyboards/hineybush/h75_singa/keymaps/wkl_std/keymap.c [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS,
+    KC_TRNS,KC_TRNS,KC_TRNS,KC_TRNS, KC_TRNS, KC_TRNS,KC_TRNS,KC_TRNS
+    ),
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    return true;
+}
+
+
```
Comment (fauxpark) [inline keyboards/hineybush/h75_singa/keymaps/wkl_std/keymap.c]: As above.

File: keyboards/hineybush/h75_singa/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# h75_singa
+
+![h75_singa](imgur.com image replace me!) - TBD
```
Comment (fauxpark) [inline keyboards/hineybush/h75_singa/readme.md]: Need an image here :)

Comment (hineybush) [reply keyboards/hineybush/h75_singa/readme.md]: The PCBs are in production, I plan to add an image once they "exist"

**Pull request merged**